odl/operator/operator.py

```diff
@@ -707,6 +707,54 @@ def __call__(self, x, out=None, **kwargs):
                     raise_from(new_exc, err)
         return out
 
+    def norm(self, estimate=False, **kwargs):
```

Comment:
At the moment I am looking at X-ray transform examples (e.g. PET, CT). Here you need to estimate the operator norm but depending on the computational cost, you may want to do it differently for each case. For example, on a small scanner you may want to run 100 power iterations, on a big one you may want to do it for 20 iterations and add a 5 % margin to it or to load stored values for it. Of course, this can be done for subclasses that really need it.